src/components/ChatHeader/ChatHeader.js

```diff
@@ -211,7 +211,7 @@ const ChatHeader = ({
       </Box>
       <Box display="flex" alignItems="center">
         {avatarUrl && (
-          <img width="20px" height="20px" src={avatarUrl} alt="avatar" />
+          <img width="20px" height="20px" src={avatarUrl} alt="avatar" style={{marginRight:"15px", borderRadius:"10px"}} />
```

Comment:
Have you checked for all screen sizes? Also, can you paste screenshots of UI for each screen. Please try these for a bigger channel name too.

Reply:
Hey @sidmohanty11 I am attaching the screenshot of the changes and it is fine for every screen size. Please review it once. And also I found a BUG in ChatHeader component that is when we use larger channel name and try to minimize the screen the ChatHeader component will overflow the screen, I am planning to make a seperate PR for this issue.
![Screenshot from 2023-02-18 19-08-51](https://user-images.githubusercontent.com/72178398/220119918-e3ccdd71-b860-4fff-b7f4-96058bf11170.png)

![Screenshot from 2023-02-18 19-09-31](https://user-images.githubusercontent.com/72178398/220119863-bac1e96f-7fce-409b-b78e-b76bae1f777c.png)